test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -729,7 +728,7 @@ public void LdaWorkoutEstimatorCore()
             builder.AddColumn("F1V", NumberDataViewType.Single, data);
             var srcView = builder.GetDataView();
 
```

Comment:
Perhaps a comment can be added here that explains the need to reset the random number generator for each received document due to way that ML.NET and LdaNative utilize multiple threads differently, for future readability.

Reply:
sure, thanks

---
In reply to: [390162926](https://github.com/dotnet/machinelearning/pull/4927#discussion_r390162926) [](ancestors = 390162926)